mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapper.java

```diff
@@ -61,6 +64,7 @@
         .count(0L)
         .unseen(0L)
         .build();
+    public static final int UPDATE_BATCH_SIZE = 200;
```

Comment:
Of course! Dichotomy from 1000 to fibd optimum.
Turned out that around 💯 was the best :-)